src/lib/core/__test__/user.unit.test.ts

```diff
@@ -249,12 +254,25 @@ describe('User module', () => {
       email: email_1,
       name: text,
       // hashedPassword: text,
+      remarks: text,
     }
     expect(async () => await createUser(param)).rejects.toThrow(
       /^missing hashedPassword$/,
     )
   })
 
+  test('Missing remarks param should return a missing remarks error', async () => {
```

Comment:
The remarks field should be optional, we don't force users to enter remarks during account creation. We can skip this test.

But, we do need a test to verify whether the assessment returns the remarks field. This is compulsory to make sure the candidate information is rendered properly.